src/new_hope/week2/day1.clj

```diff
@@ -0,0 +1,19 @@
+(ns new-hope.week2.day1
```

Comment:
Please always apply formatting to the code before checking it in (Code -> Reformat Code menu command in IntelliJ)